cmd/auth-handler.go

```diff
@@ -244,6 +234,19 @@ func getClaimsFromToken(token string) (map[string]interface{}, error) {
 		claims.MapClaims[iampolicy.SessionPolicyName] = string(spBytes)
 	}
 
+	// If LDAP claim key is set, return here.
+	if _, ok := claims.MapClaims[ldapUser]; ok {
```

Comment:
Can STS LDAP credentials have session policy ? if not let's move this before line 221 so the logic won't be overlapped

Reply:
Yes LDAP can have session policy